js_modules/dagster-ui/packages/ui-core/src/asset-graph/AssetGraphExplorer.tsx

```diff
@@ -602,3 +585,9 @@ const assetKeyTokensInRange = (
 
   return uniq(ledToTarget);
 };
+
+function normalizePathnameForCacheKey(pathname: string) {
+  const _path = pathname.endsWith('/') ? pathname.slice(0, pathname.length - 1) : pathname;
+  // Remove the op query string
+  return _path.split('~')[0];
```

Comment:
I think there may actually be more control characters in this URL -- can you the parsed `explorerPath` instead?

Reply:
Ahh I see this is trying to account for the deployment name. Let's move this helper to the same file as the explorerPath helpers so that we don't forget there's a `split('~')` with a dependency on the explorerPath format hiding in here